src/Core/Entity/Form.php

```diff
@@ -24,6 +24,7 @@ class Form extends StaticEntity
 	protected $disabled;
 	protected $created;
 	protected $updated;
+	protected $hide_author;
```

Comment:
When selected on a Survey, only users with edit permission will see the name of the survey author.